examples/widget/package.json

```diff
@@ -11,27 +11,27 @@
     "prisma:studio": "prisma studio"
   },
   "dependencies": {
-    "@clerk/nextjs": "^3",
+    "@clerk/nextjs": "^3.7.0",
     "@prisma/client": "^3.3.0",
     "clsx": "^1.1.1",
     "micro": "^9.3.4",
-    "next": "12.1.5",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "next": "12.2.2",
+    "prisma": "^3.3.0",
```

Comment:
🙃 
I think the `prisma` package is only meant for devDeps since it is the cli tool.